src/avs/AVSDetailsTVLTab.jsx

```diff
@@ -0,0 +1,274 @@
+import {
+  Spinner,
+  Table,
+  TableBody,
+  TableCell,
+  TableColumn,
+  TableHeader,
+  TableRow,
+  Image,
+  Skeleton,
+  Spacer,
+  Tooltip
+} from '@nextui-org/react';
+import React, { useMemo, useState } from 'react';
+import {
+  BEACON_STRATEGY,
+  EIGEN_STRATEGY,
+  STRATEGY_ASSET_MAPPING
+} from './helpers';
+import { useTailwindBreakpoint } from '../shared/useTailwindBreakpoint';
+import { formatUSD, formatETH, formatNumber } from '../shared/formatters';
+
+export default function AVSDetailsTVLTab({
+  totalTokens,
+  lst,
+  ethRate,
+  isAVSLoading
+}) {
+  return (
+    <>
+      {/*line chart*/}
+      <div className="bg-content1 border border-outline flex items-center justify-center h-[512px] p-4 rounded-lg w-full">
+        <Spinner color="primary" size="lg" />
+      </div>
+
+      {/*layout*/}
+      <div className="flex flex-col md:flex-row w-full h-full">
+        <div className="basis-1/2 mt-4">
+          <TokensBreakdownList
+            totalTokens={totalTokens}
+            isAVSLoading={isAVSLoading}
+            ethRate={ethRate}
+          />
+          <Spacer y={4} />
+          <LSTBreakdownList
+            lst={lst}
+            isAVSLoading={isAVSLoading}
+            ethRate={ethRate}
+          />
+        </div>
+        {/* treemap */}
+        <div className="basis-1/2 mt-4 md:ml-4">
+          <div className="bg-content1 border border-outline flex items-center justify-center h-full min-h-[512px] p-4 rounded-lg w-full">
+            <Spinner color="primary" size="lg" />
+          </div>
+        </div>
+      </div>
+    </>
+  );
+}
+
+const tokens = {
+  eth: {
+    name: 'Beacon',
+    symbol: 'ETH',
+    logo: '/eth.png'
+  },
+  lst: {
+    name: 'Liquid Staking Tokens',
+    symbol: 'ETH',
+    logo: '/eth.png'
+  },
+  eigen: {
+    name: 'Eigen',
+    symbol: 'EIGEN',
+    logo: '/eigen.webp'
+  }
+};
+
+function TokensBreakdownList({ totalTokens, isAVSLoading, ethRate }) {
+  const sortedTotalTokens = useMemo(() => {
+    const arr = Object.entries(totalTokens);
+    arr.sort((a, b) => b[1] - a[1]);
+    return arr;
+  }, [totalTokens]);
+
+  const sum = useMemo(
+    () => sortedTotalTokens.reduce((acc, [_, total]) => acc + total, 0),
+    [sortedTotalTokens]
+  );
+
+  const compact = !useTailwindBreakpoint('md');
+
+  return (
+    <Table
+      aria-label="Breakdown of ETH, EIGEN and Liquid Staking Tokens"
+      layout="fixed"
+      hideHeader
+      classNames={{
+        wrapper: 'border border-outline rounded-lg',
+        tr: 'border-b border-outline last:border-none'
+      }}
+      topContent={
+        <div className="text-foreground-1 text-medium">Tokens distribution</div>
+      }
+    >
+      <TableHeader>
+        <TableColumn className="w-1/2">Token</TableColumn>
+        <TableColumn>Total</TableColumn>
+      </TableHeader>
+      <TableBody>
+        {isAVSLoading &&
+          [...new Array(3)].map((_, i) => (
```

Comment:
Doesn't this work just the same? :)

```suggestion
          new Array(3).map((_, i) => (
```